lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -31,6 +31,12 @@
     required: false
     default: "false"
     choices: [ "true", "false" ]
+  timeout_in_minutes:
```

Comment:
I'd prefer the name `create_timeout` since this only applies on create, and I wouldn't want users to mix it up with how long the module itself waits for the stack, but is a stack-level timeout.

Reply:
Done